plugin/file/metrics.go

```diff
@@ -0,0 +1,53 @@
+package file
+
+import (
+	"github.com/coredns/coredns/plugin"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// Metrics for the secondary plugin
+var (
+	UpdatedSeconds = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: plugin.Namespace,
+		Subsystem: "secondary",
+		Name:      "last_updated_seconds",
```

Comment:
Name seems confusing (reads like a timestamp of last update). `last_update_dur_sec`?

Reply:
```suggestion
		Name:      "update_duration_seconds",
```